beetsplug/lyrics.py

```diff
@@ -705,6 +748,17 @@ def __init__(self):
         self.backends = [self.SOURCE_BACKENDS[source](self.config, self._log)
                          for source in sources]
 
+    def sanitize_bs_sources(self, sources):
+        for source in self.BS_SOURCES:
+            if source in sources:
+                self._log.warning(u'To use the %s lyrics source, you must '
```

Comment:
This should be a `debug` rather than a `warning` call. This is important so that there aren't spurious warnings when the user first enables the plugin—they should get the default set of non-BS sources working without any messages.

It was a warning for the Google backend because it needs to be enabled explicitly. It seems… OK, I suppose?… to leave it as a debug message for simplicity…